inc/mailcollector.class.php

```diff
@@ -967,12 +967,8 @@ function buildTicket($uid, \Laminas\Mail\Storage\Message $message, $options = []
          $tkt['date'] = $headers['date'];
       }
 
-      // Detect if it is a mail reply
-      $glpi_message_match = "/GLPI-([0-9]+)\.[0-9]+\.[0-9]+@\w*/";
-
-      // Check if email not send by GLPI : if yes -> blacklist
-      if (!isset($headers['message_id'])
-          || preg_match($glpi_message_match, $headers['message_id'], $match)) {
+      // Blacklist emails that are not matching mailcollector UUID
+      if (!$this->isMessageMatchingUuid($message)) {
```

Comment:
This may be a problem.

Pro: GLPI should not collect a message that containes a reference to another GLPI item.

Cons: It will not be possible to automatically create an issue in a GLPI `X` by sending it a notification from a GLPI `Y`.

Reply:
"Cons: It will not be possible to automatically create an issue in a GLPI X by sending it a notification from a GLPI Y." I'm not sure this is something we really want to deal with; this should cause many issues

Reply:
Fixed.